var/spack/repos/builtin/packages/py-pyzmq/package.py

```diff
@@ -26,8 +26,9 @@ class PyPyzmq(PythonPackage):
     version('16.0.2', sha256='0322543fff5ab6f87d11a8a099c4c07dd8a1719040084b6ce9162bcdf5c45c9d')
     version('14.7.0', sha256='77994f80360488e7153e64e5959dc5471531d1648e3a4bff14a714d074a38cc2')
 
-    depends_on('python@2.7:2.8,3.3:', type=('build', 'run'), when='@18:')
-    depends_on('python@3.6:', type=('build', 'run'), when='@22.3.0:')
+    # Python 3.9 build issues
```

Comment:
I'm not actually sure why this is the case. It builds fine with Python 3.9 on my laptop.